doc/man3/X509_cmp_time.pod

```diff
@@ -2,24 +2,31 @@
 
 =head1 NAME
 
-X509_cmp_time, X509_cmp_current_time, X509_time_adj, X509_time_adj_ex
+X509_cmp_time, X509_cmp_current_time, X509_cmp_timeframe,
+X509_time_adj, X509_time_adj_ex
 - X509 time functions
 
 =head1 SYNOPSIS
 
  int X509_cmp_time(const ASN1_TIME *asn1_time, time_t *in_tm);
  int X509_cmp_current_time(const ASN1_TIME *asn1_time);
+ int X509_cmp_timeframe(const X509_VERIFY_PARAM *vpm,
+                        const ASN1_TIME *start, const ASN1_TIME *end);
```

Comment:
Can we add a HISTORY section to this file noting that the function was added in OpenSSL 3.0?

Reply:
Sure, done.